cmd/escape_password.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"errors"
+	"net/url"
+
+	"github.com/spf13/cobra"
+)
+
+func EscapePasswordCommand() *cobra.Command {
+	var inputPassword string
+	cmd := &cobra.Command{
+		Use:   "escape-password",
+		Short: "Utility to help escape passwords with special characters.",
+		Long:  `Utility to help percent-encode passwords that have special characters.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.New("received empty password string")
+			}
+			passwordStr := args[0]
+			cmd.Printf("Substitute the following string in your original connection url string:\n%s\n", url.QueryEscape(passwordStr))
```

Comment:
I assume this is just returning the encoded password and not the whole connection string?

```suggestion
			cmd.Printf("Substitute the following encoded password in your original connection string:\n%s\n", url.QueryEscape(passwordStr))
```

Reply:
Yep, it's just the encoded password, not the string.

Reply:
I'll take this suggestion.